virtualization/windowscontainers/container-networking/advanced.md

```diff
@@ -96,15 +90,26 @@ We're working on platform changes to automatically detect/prevent this issue. Cu
 C:\> reg delete HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\Tcpip6\Parameters  /v DisabledComponents  /f
 ```
 
+
+#### Linux Containers on Windows
+
+**NEW:** We are working to make it possible to run Linux and Windows containers side-by-side _without the Moby Linux VM_. See this [blog post about Linux Containers on Windows (LCOW)](https://blog.docker.com/2017/11/docker-for-windows-17-11/) for details.
+
 #### Moby Linux VMs use DockerNAT switch with Docker for Windows (a product of [Docker CE](https://www.docker.com/community-edition)) instead of HNS internal vSwitch 
+
 Docker for Windows (the Windows driver for the Docker CE engine) on Windows 10 will use an Internal vSwitch named 'DockerNAT' to connect Moby Linux VMs to the container host. Developers using Moby Linux VMs on Windows should be aware that their hosts are using the DockerNAT vSwitch rather than the vSwitch that is created by the HNS service (which is the default switch used for Windows containers).
```

Comment:
I think you should add that LCOW (Linux Containers on Windows) will not use the DockerNAT vSwitch but rather the normal, "nat" vSwitch created by default. If users are not using LCOW but instead continuing to use Moby Linux VM to run Linux containers, then you are correct - it will use a special DockerNAT vSwitch.